unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/IRPCSocialApiBridge.cs

```diff
@@ -0,0 +1,72 @@
+using Cysharp.Threading.Tasks;
+using DCl.Social.Friends;
+using DCL.Social.Friends;
+using Decentraland.Renderer.RendererServices;
+using Decentraland.Social.Friendships;
+using RPC;
+using System;
+using System.Threading;
+
+namespace MainScripts.DCL.Controllers.FriendsController
+{
+    public interface IRPCSocialApiBridge
+    {
+        event Action<UserStatus> OnFriendAdded;
+        event Action<string> OnFriendRemoved;
+        event Action<FriendRequest> OnFriendRequestAdded;
+        event Action<string> OnFriendRequestRemoved;
+        event Action<AddFriendRequestsPayload> OnFriendRequestsAdded;
+        event Action<FriendshipUpdateStatusMessage> OnFriendshipStatusUpdated;
+        event Action<FriendRequestPayload> OnFriendRequestReceived;
+
+        UniTaskVoid InitializeClient(CancellationToken cancellationToken = default);
+
+        UniTaskVoid InitializeSubscriptions(CancellationToken cancellationToken = default);
+
+        UniTask<FriendshipInitializationMessage> InitializeFriendshipsInformation(CancellationToken cancellationToken = default);
+
+        UniTask ListenToFriendEvents(CancellationToken cancellationToken = default);
+
+        UniTask RejectFriendship(string friendRequestId, CancellationToken cancellationToken = default);
+
+        UniTask<FriendRequest> RequestFriendship(string friendUserId, string messageBody, CancellationToken cancellationToken = default);
+
+        UniTask UpdateFriendship(UpdateFriendshipPayload updateFriendshipPayload, CancellationToken cancellationToken = default);
+
+        void RemoveFriend(string userId);
```

Comment:
i think you should remove all the `void` calls that should be deprecated by async `UniTask` operations

Reply:
Removed all void methods from the interface, and clarified with a comment that all of the remaining will be implemented in the future.